app-moodle-default-release/init.sls

```diff
@@ -0,0 +1,21 @@
+#
+# Ubiquitous Moodle
+#
+# @author Luke Carrier <luke@carrier.im>
+# @copyright 2016 Luke Carrier
+#
+
+include:
+  - app-default-release
```

Comment:
I wonder how the role worked before the separation as it was missing it :thinking: